src/editors/AdvancedEditor.tsx

```diff
@@ -43,12 +47,19 @@ const AdvancedEditor = ({ usageKey, onClose }: AdvancedEditorProps) => {
   }, []);
 
   return (
-    <EditorModalWrapper onClose={onClose as () => void}>
-      <LibraryBlock
-        usageKey={usageKey}
-        view="studio_view"
+    <>
+      <EditorModalWrapper onClose={openCancelConfirmModal}>
```

Comment:
I assume that we don't have a simple way to detect whether any setting has changed (to skip showing confirmation if the data is not dirty). I did check `event.data` and could not find anything.

Reply:
Yes, since it is an iframe, it is very difficult to know if the data is dirty. There is also a precedent for this in the Video Block that always opens the popup when the modal is closed.